server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/version/v60/PopulateUuidColumnsOfProjects.java

```diff
@@ -55,8 +55,8 @@ public void execute(Context context) throws SQLException {
     Map<Long, String> componentUuidById = new HashMap<>();
     // rootId for root nodes (ie. column root_id is null)
     context.prepareSelect("select distinct p1.id, p1.uuid from projects p1" +
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "select distinct p1.id, p1.uuid from projects p1" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)